critiquebrainz/db/users_test.py

```diff
@@ -0,0 +1,32 @@
+from critiquebrainz.data.testing import DataTestCase
+from critiquebrainz.data.model.user import User
+from critiquebrainz.db.users import avatar, get_many_by_mb_username
+from hashlib import md5
+
+
+class UserTestCase(DataTestCase):
+    def setUp(self):
+        super(UserTestCase, self).setUp()
+        self.user1 = User.get_or_create("test", musicbrainz_id='0')
+        self.user2 = User.get_or_create("test1", musicbrainz_id='1')
+
+    def test_get_many_by_mb_username(self):
+        users = [(self.user1.display_name, self.user1.musicbrainz_id),
+                 (self.user2.display_name, self.user2.musicbrainz_id)]
+        dbresults = get_many_by_mb_username([self.user1.display_name,
+                                             self.user2.display_name])
```

Comment:
`display_name` is not the same thing as `musicbrainz_id` (MusicBrainz username).